### PR TITLE
Fixes #84

### LINKED ISSUE #84: 1-1-stable, :review breaks collection of payment
For the 1-1-stable branch, enabling :review breaks collection of payment.

Order moves to :complete, but no actual payment is collected from the user. Just deployed 1-1-3 to production, ask me how I know...

Disabling :review moves things along nicely. Don't forget to re-enter password.

### PR DESCRIPTION
**Pull request by fmd**
This pull request changes the order details deface. Previously the file was still referring to `order.payment` in two places, even after https://github.com/spree/spree_paypal_express/commit/020d519357d0c68bb11acd525ba2a2f8c316dc22
This fix isn't particularly graceful, as I'm new to ruby, but it does the trick; i.e. after paypal redirects back to my spree store, it doesn't show an error page saying that it can't find the order.payment method.

### CODE REVIEW

**Comment by austin5513** [comment]
Yes, I encountered the same issue as you did, thanks for fix.

File: spree_paypal_express.gemspec [hunk 1]
```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.has_rdoc      = false
 
-  s.add_dependency 'spree_core', '~> 1.3.0'
+  s.add_dependency 'spree_core', '~> 1.3.x'
```
Comment (JDutil) [inline spree_paypal_express.gemspec:16]: I dont think using x here is valid it should remain 0 or changed to the last stable release which is 2 I think.

**Comment by fmd** [comment]
the pull request is for 1-3-stable, so surely changing it to 2 wouldn't be right? I changed it to 1.3.x because my bundle update was giving me a hard time as 1-3-stable branch of spree is no longer 1.3.0. I can resubmit the pull request without this commit if you'd prefer.

**Comment by msevestre** [comment]
@fmd: Yes please could you resubmit the PR using '~> 1.3'
@jdutil : Do you think you could merge that PR? That would be awesome

Thanks guys!

**Comment by JDutil** [comment]
I've added to master and 1-3-stable.

**Comment by AlexEscalante** [comment]
Working great! Many thanks...

**Pull request closed without merging**